src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -568,6 +568,7 @@ MulticopterPositionControl::parameters_update(bool force)
 		param_get(_params_handles.thr_min, &_params.thr_min);
 		param_get(_params_handles.thr_max, &_params.thr_max);
 		param_get(_params_handles.thr_hover, &_params.thr_hover);
+		_params.thr_hover = math::constrain(_params.thr_hover, _params.thr_min, _params.thr_max);
 		param_get(_params_handles.alt_ctl_dz, &_params.alt_ctl_dz);
```

Comment:
is it a constrain? I think first it needs to be checked that params.thr_min is indeed smaller than param.thr_max. And if para.thr_hover is larger than param.thr_max and param.thr_min, do we really want to set it to max? I would say the value param.thr_hover is then just set incorrectly and I would set it to the mean of max and min